### PR TITLE
[video] Make `showNowPlayingNotification` default to `false`

### PR DESCRIPTION
**Pull request by behenate**
# Why

We shouldn't display the notification by default. Especially important for smaller videos.

# How

On android needed to do some changes, turns out the property wasn't properly applied on first load. On iOS it already defaulted to false.

# Test Plan

Tested in BareExpo on iOS and Android

### CODE REVIEW

File: packages/expo-video/src/VideoPlayer.types.ts [hunk 1]
```diff
@@ -108,6 +108,10 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
 
   /**
    * Boolean value determining whether the player should show the now playing notification.
+   *
+   * @default false
+   * @platform ios
+   * @platrorm android
```
Comment (tsapeta) [inline packages/expo-video/src/VideoPlayer.types.ts:114]: As per the [documentation writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms), Android should go first
```suggestion
   * @platform android
   * @platrorm ios
```

**Review by tsapeta — APPROVED** [review]

**Review by amandeepmittal — APPROVED** [review]

**Comment by tslater** [comment]
@behenate I'm hoping this fixes issues we're having where changing showNowPlayingNotification to `true` does not disable the notification. In fact, the notification stacks for each video we play and persists when we hard close the app. We're getting bad reviews, with multiple users calling our app a "virus" because of this behavior. Do you think this will fix it? What's the timeline for next release?

**Comment by behenate** [comment]
@tslater The changes are already on the `sdk-51` branch, so you can expect a new `expo-video` version within a week or two. The notification not disappearing is concerning, it should close when the app is hard-closed and it does in our test app. You shouldn't be getting a new notification for each video you play (unless you intentionally had multiple players with `showNowPlayingNotification` set to `true`), you should check if the players are getting deallocated after you stop using them.

**Comment by tslater** [comment]
@behenate I meant to say that setting showNowPlayingNotification to `false` does not hide the notification. There's no way to get rid of it on Android. Are you not seeing that? We can try and make a minimal reproduction of the issue.

**Comment by behenate** [comment]
@tslater Oh ok, I did notice that while doing this PR, it's fixed now. Nevertheless you shouldn't have multiple notifications unless you are playing multiple videos at once. Having multiple notifications means that you have multiple players = wasted memory.

**Pull request merged**